### DISCUSSION TITLE
webargs usage (parsing arguments from query string or json)

### DISCUSSION THREAD

**Discussion by cpvandehey** [pallets/quart#394] [Q&A]
Hello Quart folks :)

I have been using the [webargs library](https://webargs.readthedocs.io/en/latest/index.html#) for parsing request args (_from query strings or json_) in my flask projects. 

Sadly, It appears that webargs doesn't have support for Quart. 

With that said, I did find a gist that gives a fairly [dated implementation of a custom one here](https://gist.github.com/mvanderlee/8c4d09b72e4651485bd08d765eda9f27) (_and still works for the most part_).

Im curious if there are other tools available to accomplish the same thing or if there is a formal webargs parser implementation somewhere else.

Thanks for any and all tips/advice!

**Comment by catwell** [comment]
Not really the same thing but there is https://github.com/pgjones/quart-schema

**Reply by cpvandehey** [reply]
interesting! Thanks for sharing that. Its in a little bit of different domain from what im looking for.
